### PR TITLE
First version of FBUP thesis template

### PR DESCRIPTION
**Pull request by avramescudana**

### CODE REVIEW

File: FPUB thesis template/qcd.tex [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+where we introduced the dual field strength tensor as
+\begin{equation*}
+    ^*F^{\mu\nu}=\frac{1}{2}\epsilon^{\mu\nu\rho\sigma}F_{\rho\sigma}.
+\end{equation*}}
+\shadedeq{
+    \textsf{D}_\nu F^{\nu\mu}=gJ^\mu
+}
+in which $J^\mu=\sum\limits_a J^{a,\mu} t^a$ with $J^{a,\mu}=\overline{\psi}\gamma^\nu t^a\psi$ being the color current. The color current is covariantly conserved $\textsf{D}_\mu J^\mu=0$.
+
+%This is an example chapter.
```
Comment (SebastianM-C) [inline FPUB thesis template/qcd.tex]: Why is this chapter commented out?

File: FPUB thesis template/reference.bib [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+@book{schwartz,
+    author = "Schwartz, M. D.",
+    title = "{Quantum Field Theory and the Standard Model}",
+    isbn = "978-1-107-03473-0, 978-1-107-03473-0",
+    publisher = "Cambridge University Press",
+    month = "3",
+    year = "2014"
+}
```
Comment (SebastianM-C) [inline FPUB thesis template/reference.bib]:
```suggestion
}

```

Comment (SebastianM-C) [reply FPUB thesis template/reference.bib]: Add newline

File: FPUB thesis template/thesis.tex [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+\def\thesistitle{Wilson Loops}
+% In case of no subtitle, just leave this field empty {}
+\def\thesissubtitle{Mundane primer for the uninitiated}
+\def\thesistype{Bachelor}
+\def\firstsupervisor{First Supervisor}
+\def\firstsupervisorprefix{Prof. Dr.}
+% In case of no second supervsior, just leave these fields empty {}
+\def\secondsupervisor{Second Supervisor}
+\def\secondsupervisorprefix{Lect. Dr.}
+% In case of a third supervisor, well...
```
Comment (SebastianM-C) [inline FPUB thesis template/thesis.tex:40]: I think it's ok for the template to assume only 2 supervisors.

File: FPUB thesis template/qcd.tex (hunk 1 above)
Comment (avramescudana) [reply FPUB thesis template/qcd.tex]: It's a chapter from the original template, should be removed.

**Review by SebastianM-C — APPROVED** [review]
For now we can leave the pdf file committed, but after building with ci works, we can remove it.
We should also add some instructions in the README for using the template

**Pull request merged**